app/backend/src/couchers/models.py

```diff
@@ -164,6 +164,9 @@ class User(Base):
     is_deleted = Column(Boolean, nullable=False, server_default=text("false"))
     is_superuser = Column(Boolean, nullable=False, server_default=text("false"))
 
+    undelete_token = Column(String, nullable=True)
+    undelete_until = Column(DateTime(timezone=True), nullable=True)
```

Comment:
Document these. Actually I like storing all necessary information in the link better, like you did in https://github.com/Couchers-org/couchers/pull/2475 . Less state to keep track of in the database.

Reply:
These things are good to keep in the DB: they're quite a bit more critical and we need these for debugging and visibility!

Reply:
Documented